src/app/pages/card-import/card-import.page.ts

```diff
@@ -170,13 +177,23 @@ export class CardImportPage implements OnInit {
 
   updateCard() {
       this.cardService.updateCard(this.card).then(() => {
-      this.router.navigateByUrl('/profile');
+      this.router.navigateByUrl('/home/home/profile');
         this.showToast('Card updated');
       }, err => {
         this.showToast('There was a problem updating your card :(');
       });
   }
 
+  sendCard() {
```

Comment:
Because apparently I'm defying the Single Responsibility Principle and the card import page should be split up.

Reply:
That's less important than determining whether this Pull Request is about card exchange, card import, screen orientation, or formatting. This does not match what we've discussed. We should be sending `this.id`, at most, to determine which card should add the transmitted new holder uid to its holder list (this will happen in the exchange page). This can be removed until that feature is ready for work.

Reply:
It's quite the mental workout bouncing back and forth between creative storytelling and the codification of it. Wires get crossed a lot. But hey, that's what the team's for right?